tests/storage/snapshot.go

```diff
@@ -383,7 +383,11 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 			}
 
 			callVeleroHook := func(vmi *v1.VirtualMachineInstance, annoContainer, annoCommand string) (string, string, error) {
-				pod := tests.GetPodByVirtualMachineInstance(vmi)
+				var pod *corev1.Pod
+				Eventually(func() error {
```

Comment:
I think it is preferable to wait on a phase/condition over this retry.

Reply:
Done